Add INV adapter

Slokh opened the pull request:
fixes #1 

Latest report:
```
{
  "ethCallCount": 10,
  "timestamp": 1618862400,
  "block": 12272528,
  "output": {
    "WETH": {
      "balance": "5487.921142",
      "cmcId": 1027,
      "coingeckoId": "weth"
    },
    "WBTC": {
      "balance": "13.472673",
      "cmcId": 1,
      "coingeckoId": "wrapped-bitcoin"
    },
    "USDC": {
      "balance": "362867.602122",
      "cmcId": 3408,
      "coingeckoId": "usd-coin"
    },
    "DAI": {
      "balance": "6070483.111508",
      "cmcId": 4943,
      "coingeckoId": "dai"
    },
    "xsushi": {
      "balance": "117104.642497",
      "coingeckoId": "xsushi"
    },
    "yfi": {
      "balance": "103.865631",
      "coingeckoId": "yearn-finance"
    }
  }
}
```

The only thing I'm unsure about is the stabilizer calculation. Right now, I'm just using the DOLA supply in DAI. 

I also need us to confirm what is the start time we should use, for now I just picked the first anchor tx.

Attached is a zip of various reports. The CSV generation doesn't work.
[tvl.zip](https://github.com/InverseFinance/DeFi-Pulse-Adapters/files/6353766/tvl.zip)

Slokh committed:
add inverse.finance

JackieXu commented on projects/inverse/index.js:
```diff
@@ -0,0 +1,205 @@
+  Modules
+  ==================================================*/
+
+const _ = require("underscore");
```
I don't think `underscore` 's performance gains are usable if the data is gathered via a Node process, but I'm not sure how DeFi Pulse uses this module exactly.

In general, I'd say to prefer the native calls over `underscore` ones for clarity. I don't mind the `underscore` implementation, though. Can stay.

Slokh replied:
The rest of the projects all use underscore, so I tried to just stick with the same patterns to just keep it simple for reviewers.

JackieXu commented on projects/inverse/index.js:
```diff
@@ -0,0 +1,205 @@
+  const tokens = await getAllTokens(block);
+  const [allUnderlying, cashes] = await Promise.all([
+    getAllUnderlying(block, tokens),
+    getCashes(block, tokens),
```
Just a check: how do other lending platforms calculate their TVL?

This current implementation gives the number that's currently sitting in Anchor available to borrow, not what was deposited as collateral. 

The latter would be something in the order of: 
 1. Get total supply of market tokens
 2. Multiply by `exchangeRateStored` to get the number of underlying tokens
 3. Format decimals correctly

TSnark replied:
I'm not sure I agree with the above. Getting tvl on a cash basis tells us the net balance considering borrowings that are currently out of the system. By that I mean, if something is borrowed shouldn't be counted as locked.

nourharidy replied:
Agreed with @TSnark, TVL should be amount of available liquidity, not total supplied amount

JackieXu commented on projects/inverse/index.js:
```diff
@@ -0,0 +1,205 @@
+module.exports = {
+  name: "Inverse",
+  token: "INV",
+  category: "lending",
```
Are we "lending" or "assets"? I'm leaning towards "assets". Probably better answered by @nourharidy.

TSnark replied:
It should be assets as we're not purely lending, by looking at the website assets is like a wildcard (see badger).

nourharidy replied:
Agreed about assets.

Slokh replied:
Yeah I was leaning between the two. Will change to assets.

JackieXu reviewed (commented):
Looks good! 

I think it's fine to calculate consider the Dola:Dai ratio 1:1 .

I've added questions regarding calculating TVL and the category to be placed under.

nourharidy commented on projects/inverse/index.js:
```diff
@@ -0,0 +1,205 @@
+    await sdk.api.abi.call({
+      block,
+      target: dola,
+      abi: abi["totalSupply"],
```
Not all of the DOLA `totalSupply` originates from the Stabilizer. Some originate from Anchor. Instead, you should query `Stabilizer.supply()` which will return the amount of Dai stored within the Stabilizer.

nourharidy replied:
Also something to note is that DOLA in general should not be counted as part of the TVL since it's a synthetic asset that can be minted in any quantity within our system.

Slokh replied:
Yeah that's why I wasn't sure what to use. I was using `Stabilizer.supply()` initially before I changed it to the `totalSupply`.

Slokh committed:
Update stabilizer TVL calculation

Slokh commented:
Updated the stabilizer TVL calculation to use `Stabilizer.supply()` and changed category to `assets`

New TVL files are attached.
[tvl.zip](https://github.com/InverseFinance/DeFi-Pulse-Adapters/files/6353467/tvl.zip)

Slokh committed:
Set inverse start to vaults start

Slokh commented:
Set start to vaults launch. Added logic to only include anchor TVL after anchor launch block #.

New TVL files
[tvl.zip](https://github.com/InverseFinance/DeFi-Pulse-Adapters/files/6353771/tvl.zip)

ghost commented:
Merged upstream
